postman/p4pa-auth-E2E.postman_collection.json

```diff
@@ -919,6 +918,62 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "getClients",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Auth - getClients - Responses with 200\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});\r",
+							"\r",
+							"pm.test(\"Auth - getClientSecret - Verify response body\", function () {\r",
+							"    let jsonResponse = pm.response.json();\r",
+							"\r",
+							"    pm.expect(jsonResponse).to.be.an(\"array\").that.is.not.empty;\r",
+							"    jsonResponse.forEach(dto => {\r",
+							"    pm.expect(dto).have.property(\"firstName\").to.eq(\"IPA_TESTSERVICE001\")\r",
+							"    pm.expect(dto).have.property(\"clientName\").to.eq(\"SERVICE001\")\r",
+							"    pm.expect(dto).have.property(\"organizationIpaCode\").to.eq(\"IPA_TEST\")\r",
+							"    pm.expect(dto).have.property(\"clientSecret\")\r",
```

Comment:
```suggestion
							"    pm.expect(dto).not.have.property(\"clientSecret\")\r",
```

Reply:
execute the application locally, then run the postman collection! this should fail without this fix

Reply:
you beat me to it, I was correcting the test